### DISCUSSION TITLE
Java Package Installation Error while creating Ubuntu 2204 build image in Azure.

### DISCUSSION THREAD

**Discussion by SRA91** [actions/virtual-environments#9473] [General questions] [answered]
Hi Team, While creating the Ubuntu 2204 build image got the error message 
    '"/usr/lib/jvm/temurin-21-jdk-amd64/bin/java" -version' has finished with exit code
    azure-arm.build_image:        /usr/bin/bash: line 1: /usr/lib/jvm/temurin-21-jdk-amd64/bin/java: No such file or directory
    azure-arm.build_image:    at "`"$javaPath`" -version" | Should -ReturnZeroExitCode, /imagegeneration/tests/Java.Tests.ps1:43
    azure-arm.build_image:    at <ScriptBlock>, /imagegeneration/tests/Java.Tests.ps1:43
    the execution stopped at this point.

**Comment by SRA91** [comment]
![ImageCreationError](https://github.com/actions/runner-images/assets/160568789/5099e9c4-1fe4-4d1a-9169-08537bee6f4a)

**Comment by erik-bershel** [comment, marked as answer]
Hey @SRA91!
It's a common availability issue of the Adoptium repository. Just wait couple hours and try again - it should be fixed. We tired of this too I may say.
